src/task2/task2.controller.ts

```diff
@@ -0,0 +1,52 @@
+import {

```

Comment:
try to name your controller based on the module -> Flight, Customer, Booking which will be more meaningful rather as task2